README.md

```diff
@@ -25,7 +25,8 @@ FAUCET_SECRET_PHRASE=<secret phrase>
 
 # Faucet send amount
 TESTNET_FAUCET_AMOUNT = '15'
-MAINNET_FAUCET_AMOUNT = '0.002'
+SHIDEN_FAUCET_AMOUNT = '0.001'
+ASTAR_FAUCET_AMOUNT = '0.002'
```

Comment:
Any reason why Astar has more than Shiden? I thought it should be the other way around.

Reply:
Ok, I'll change it to 0.002.